examples/get-artist-albums/main.bal

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+import ballerinax/spotify;
+
+configurable string token = ?;
+spotify:Client spotify = check new ({
+    auth: {
+        token
+    }
+});
+
+final string artistId = "0TnOYISbd1XYRBk9myaseg";
+
+public function main() returns error? {
+    PagingArtistDiscographyAlbumObject = check spotify->/{artistId}/albums;
+}
```

Comment:
@Sachindu-Nethmin we can improve this example to 
1. retrieve the artistId from the user (as a main function argument) and extract the details based on that
2. print the related info (e.g. artist name, number of albums, the list of albums with the released dates, etc)

Reply:
Agreed with @NipunaRanasinghe, Let's add some processing to the example and make it complete.